docs/tex/tut_inference_networks.tex

```diff
@@ -117,31 +116,9 @@ \subsubsection{Footnotes}
 general technique that can be applied to models beyond
 probabilistic decoders.
 
-This tutorial is taken from text originating in Tran et al. (2016).
+This tutorial is taken from text originating in \citet{tran2015variational}.
```

Comment:
actually it's tran2016variational. :) (the bib is [here](https://github.com/blei-lab/publications/blob/master/2016_TranRanganathBlei/TranRanganathBlei_2016.bib))